### PR TITLE
Updated README

### LINKED ISSUE #7: Update Readme
This readme needs more details to describe the current state of the repo and give more context of the project in general.

### PR DESCRIPTION
**Pull request by sammyshj**
Standardised README with the other Privly browser extensions
Fixes #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+## About ##
+
+Privly is a developing set of browser extensions for protecting content wherever it is posted on the internet. This extension allows users to view content on any website, without the host site being able to read the content. For more information on what Privly is, [read about us](https://priv.ly/pages/about).
+
+## Development Status ##
+
+**Alpha Version**
```
Comment (smcgregor) [inline README.md]: This is not yet an alpha version.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+[![Build Status](https://travis-ci.org/privly/privly-safari.svg)](https://travis-ci.org/privly/privly-safari)
```
Comment (smcgregor) [inline README.md]: This is not hooked into Travis yet.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+The Privly Safari extension is currently maintained by [Sean McGregor](https://github.com/smcgregor) and [Sambuddha Basu](https://github.com/sammyshj), on behalf of the [Privly Foundation](http://www.privly.org).
```
Comment (smcgregor) [inline README.md]: No maintainers, yet. You would become a maintainer when we declare an Alpha.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+## About this Extension ##
+
+This extension currently supports:
+
+* Locally served posting and injectable applications.
+* All the implemented [Injectable Applications](https://github.com/privly/privly-organization/wiki/Injectable-Applications) of the content server.
+
+## Testing/Submitting Bugs ##
+
+Extension integration test cases are found at [test.privly.org](http://test.privly.org). If you have discovered a bug, only [open a public issue](https://github.com/privly/privly-safari/issues/new) on GitHub if it could not possibly be a security related bug. If the bug affects the security of the system, please report it privately at [privly.org](http://www.privly.org/content/bug-report). We will then fix the bug and follow a process of responsible disclosure.
```
Comment (smcgregor) [inline README.md]: The bug reporting practices changed for security bugs. Follow the link for the current details.

File: README.md (hunk 1 above)
Comment (sammyshj) [inline README.md]: Should this be changed to pre-alpha version?
Or just under development?

Comment (smcgregor) [inline README.md]: Pre-alpha is good.

**Comment by sammyshj** [comment]
Should I squash my commits if the changes look okay?

**Comment by smcgregor** [comment]
Please do. :)

**Pull request merged**